models/User.model.js

```diff
@@ -3,10 +3,13 @@ const Schema = mongoose.Schema;
 
 const userSchema = Schema(
   {
+    userId: Number,
+    petitions: { type: Schema.ObjectId, ref: "Petition" },
```

Comment:
If I'm not mistaken these two should be an array because we need users to be able to create many `petitions` & `participations`.

`[{ type: Schema.Types.ObjectId, ref: "Petition" }]`

`[{ type: Schema.Types.ObjectId, ref: "Paticipant" }]`